src/core/ConfigManager.cpp

```diff
@@ -673,3 +697,25 @@ void ConfigManager::initDevelopmentWorkingDir()
 		cmakeCache.close();
 	}
 }
+
+// If configversion is not present, we will convert the LMMS version to the appropriate
+// configuration file version for backwards compatibility.
+const unsigned int ConfigManager::legacyConfigVersion()
+{
+	ProjectVersion createdWith = m_version;
+
+	createdWith.setCompareType(ProjectVersion::Build);
+
+	if( createdWith < "1.1.90" )
+	{
+		return 0;
+	}
+	else if( createdWith < "1.1.91" )
+	{
+		return 1;
+	}
+	else
+	{
+		return 2;
+	}
+}
```

Comment:
I'm not sure whether this should be changed to match the logic in `DataFile::legacyConfigVersion` or not. What do you think?

Reply:
We discussed this on Discord and the decision was to leave it as is.